persistence/schema/gov.go

```diff
@@ -1,442 +1,145 @@
-		hex.EncodeToString(params.ValidatorMaxPausedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaximumMissedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaxEvidenceAgeInBlocksOwner),
-		hex.EncodeToString(params.ProposerPercentageOfFeesOwner),
-		hex.EncodeToString(params.MissedBlocksBurnPercentageOwner),
-		hex.EncodeToString(params.DoubleSignBurnPercentageOwner),
+	fields := "name,height,value,enabled,type"
 
-		hex.EncodeToString(params.MessageDoubleSignFeeOwner),
-		hex.EncodeToString(params.MessageSendFeeOwner),
-		hex.EncodeToString(params.MessageStakeFishermanFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeFishermanFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeFishermanFeeOwner),
-		hex.EncodeToString(params.MessagePauseFishermanFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseFishermanFeeOwner),
-		hex.EncodeToString(params.MessageFishermanPauseServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageTestScoreFeeOwner),
-		hex.EncodeToString(params.MessageProveTestScoreFeeOwner),
-		hex.EncodeToString(params.MessageStakeAppFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeAppFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeAppFeeOwner),
-		hex.EncodeToString(params.MessagePauseAppFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseAppFeeOwner),
-		hex.EncodeToString(params.MessageStakeValidatorFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeValidatorFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeValidatorFeeOwner),
-		hex.EncodeToString(params.MessagePauseValidatorFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseValidatorFeeOwner),
-		hex.EncodeToString(params.MessageStakeServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessagePauseServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageChangeParameterFeeOwner),
-		DefaultBigInt,
-	)
-}
+	subQuery := fmt.Sprintf(`SELECT %s`, fields)
+	//TODO (@deblasis): clarify if and how we should use `enabled` here
+	subQuery += fmt.Sprintf(` FROM %s WHERE name='%s' AND height=%d`, ParamsTableName, paramName, height)
 
-func GetParamQuery(paramName string) string {
-	return fmt.Sprintf(`SELECT %s FROM %s WHERE end_height=%d`, paramName, ParamsTableName, DefaultBigInt)
+	return fmt.Sprintf(`INSERT INTO %s(%s) %s`, ParamsTableName, fields, subQuery)
 }
 
-func GetParamNames() (paramNames []string) {
-	paramNames = make([]string, len(SQLColumnNames))
-	copy(paramNames, SQLColumnNames)
-	return
+func ClearAllGovQuery() string {
+	return fmt.Sprintf(`DELETE FROM %s`, ParamsTableName)
 }
 
-func NullifyParamsQuery(height int64) string {
-	return fmt.Sprintf(`UPDATE %s SET end_height=%d WHERE end_height=%d`, ParamsTableName, height, DefaultBigInt)
+type govParamMetadata struct {
+	PoktValType  string
```

Comment:
We use `PoktVal` and `Property`, but should we be consistent here like `PropertyType` and `PropertyName`, for example?